src/components/player.js

```diff
@@ -452,18 +465,67 @@ export default class Player extends PureComponent {
     );
   }
 
+  renderExternalVideo(active) {
+    //if (!this.layout.hasScreenshare()) return null;
+
+    const {
+      intl,
+      data,
+    } = this.props;
+
+    const { time } = this.state;
+    const { external_videos } = data;
+
+    if (!external_videos) {
+      return;
+    }
+
+    let currentDataIndex = getCurrentDataIndex(external_videos, time);
+
+    if (currentDataIndex === -1) {
+      currentDataIndex = 0;
+    }
+
+    const video = external_videos[currentDataIndex];
+     
+    if (!video) {
+      return
+    }
+
+    const url = video.url
+    const events = video.events;
+
+     let primaryPlaybackRate = 1;
+
+    if (this.player.video)  {
+       primaryPlaybackRate = this.player.video.playbackRate();
+   }
+         
+    return (
+      <ExternalVideoPlayer 
+          active={active}
+          intl={intl}
+          videoUrl={url}
+          onPlayerReady={this.handlePlayerReady}
+          events={events}
+          primaryPlaybackRate={primaryPlaybackRate}
```

Comment:
send the function to the component.
`getCurrentPlayerTime={this.getCurrentPlayerTime}`